backend/tests/test_directml_iris.py

```
@@ -0,0 +1,179 @@
+            import torch_directml
+
+            if torch_directml.device_count() == 0:
+                pytest.skip("No DirectML devices available")
+
+            device = torch_directml.device(0)
+            # Create and delete tensors to check memory cleanup
+            for _ in range(5):
+                x = torch.randn(1000, 1000, device=device)
+                del x
+
+            logger.info(f"✓ DirectML memory management OK")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)